### DISCUSSION TITLE
Is server go implementation of socket.io https://github.com/googollee/go-socket.io safe for production?

### DISCUSSION THREAD

**Discussion by Bubuioc** [socketio/socket.io#5260] [Q&A]
I saw it is archived.Does it mean that is no more maintained, and could be older version than Node implementation?

**Comment by stn-ptr** [comment]
It is an old version, according to the readme it supports socket.io 1.4. And it is not maintained anymore.

While we are at the topic: @darrachequesne would it be possible to create a clone of [googollee/go-socket.io](https://github.com/googollee/go-socket.io) in the socketio organization so it can be maintained here? And maybe rename it to socket.io-go to conform to the other repose or even socketgo so it doesn't need to be aliased anymore (although admittedly that would break conformity).

**Reply by darrachequesne** [reply]
That's a great question.

It seems there is a fork here (https://github.com/feederco/go-socket.io), but it doesn't seem very active, and only supports the socket.io v1 protocol.

I don't have a lot of bandwidth right now to update/maintain a golang server implementation though.

**Comment by zishang520** [comment]
Hello, welcome to use:
server: https://github.com/zishang520/socket.io
client: https://github.com/zishang520/socket.io-client-go
